releasenotes/notes/metrics-api-5fb1d572d7f7137e.yaml

```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    The ``collect_metrics`` argument of the ``tracer.configure`` method has been
```

Comment:
When I made my previous comment I had assumed that runtime metrics were already publicly available but they're not! I wonder if we should remove the changelog entry for this change altogether cc @brettlangdon, @majorgreys

Reply:
@brettlangdon suggested that we keep the release note but omit docs changes for now

Reply:
We can update this note then since we opted to deprecate it rather than remove it. It can be moved to the `deprecations:` section